std_msgs/Header header
string behavior_name
trajectory_msgs/JointTrajectory trajectory
nav_msgs/Path path
---
int32 result
int32 OK = 0
int32 ERROR = 1
